### ISSUE TITLE
Collection playbooks should avoid the `-` character

### ISSUE THREAD

**Issue by dkolepp** [redhat-cop/automation-good-practices#53]
Current tooling allows you to create - and build - a playbook like: `some-test-playbook.yml`, but this will not work when used to run that collection playbook:

```
$> ansible-playbook xyz.abc.some-test-playbook.yml
```

**Comment by ericzolf** [comment]
Good point, I knew this for roles, but didn't notice for playbooks. I think, we should just make one recommendation: name everything like Ansible variables (letters, numbers and underscores), and basta.
